common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -0,0 +1,159 @@
+"""Simple utility functions that operate on course metadata.
+
+Ths is a place to put simple functions that operate on course metadata. It
```

Comment:
I've made the changes you suggested, but I'm going to wait until the tests finish on Jenkins before pushing them.